analyzer/emerald/incoming.go

```diff
@@ -188,7 +187,7 @@ func extractRound(sigContext signature.Context, b *block.Block, txrs []*sdkClien
 		}
 		blockTransactionData.Raw = cbor.Marshal(txr.Tx)
 		blockTransactionData.RelatedAccountAddresses = map[string]bool{}
-		tx, err := common.VerifyUtx(sigContext, &txr.Tx)
+		tx, err := common.OpenUtxNoVerify(&txr.Tx)
 		if err != nil {
 			logger.Error("error verifying tx", "tx_index", txIndex, "err", err)
```

Comment:
```suggestion
			logger.Error("skipping malformed tx", "tx_index", txIndex, "err", err)
```

or similar. "error verifying" is outdated for sure. "malformed" might be a little too narrow, but it does seem to be the primary reason why `UpenUtxNoVerify` might error out.

Reply:
it could be malformed, e.g. if enough of the committee dishonestly allows it, or if there's a bug in the runtime that treats it as well formed.

or it could be well formed, but we have a bug in the indexer or some other mismatch with how we decode it.